main.py

```diff
@@ -39,7 +39,7 @@ def evaluate(ner_model, gold_examples, labelled_span):
 ]
 
 labelled_data = [
-    [(7, 13)],
```

Comment:
formato del vettore labelled_data: elenco di righe(file json1)
.
.
{"meta": {"ord_id": 91, "fr_id": 0}, "labels": [[483, 544, "OG-1"], [483, 501, "IV"] ...] }
{"meta": {"ord_id": 93, "fr_id": 0}, "labels": [[4417, 4421, "OG-12"], [4424, 4436, "OG-3"], ...] }

Reply:
qui si pone anche il problema di come iterare:
a livello di documento (ord_id)    relativamente a **regex-labelled-data.json1**
Lo stesso ord_id è in **meta** nel file **annotated_first_100-doc.json1**